Makefile

```diff
@@ -29,16 +29,15 @@ prep:
 run:
 	go run ./cmd/pbapi
 
-.PHONY: generate-clients
-generate-clients:
-	generate-ib-client
-
-.PHONY: generate-ib-client
-generate-ib-client:
-	cd internal/clients/image_builder
-	wget https://raw.githubusercontent.com/osbuild/image-builder/main/internal/v1/api.yaml
+.PHONY: update-clients
+update-clients:
+	wget -O ./configs/ib_api.yaml -qN https://raw.githubusercontent.com/osbuild/image-builder/main/internal/v1/api.yaml
+
+generate-clients: internal/clients/image_builder/client.gen.go
+
+internal/clients/image_builder/client.gen.go: configs/ib_config.yaml configs/ib_api.yaml
```

Comment:
Why is that written twice?

Reply:
it is a specification of make target, so this file is a target, that should get created here and depends on those two files (if those change, makefile will know to re-make the target. It's quite cool, but bit messy xD

Reply:
but how is this file created in the first place?

Reply:
right, you're correct, this will probably not create the file, as we do not redirect the output.

Reply:
It does create the file, I do not file PRs which do not work (unless stated in the description) :-) The configuration YAML actually specifies the output directory.

@adiabramovitch yeah this is how the ancient GNU Make (or any other makefile software) treats dependencies and there is no other way to do it. It typically looks nicer, granted, our path is long and we can make it nicer with variables later on.